client/themes/crypto.css

```diff
@@ -65,14 +65,6 @@ a:hover,
 	background: #00ff0e;
 }
 
-.btn-reconnect {
-	background: #f00;
-	color: #fff;
-	border: 0;
-	border-radius: 0;
-	margin: 0;
-}
-
```

Comment:
Don't themes need to define their own values for `#connection-error`?

Reply:
I moved it to main style file.

Reply:
Alright. They were in the theme files on purpose for future-proof theming, but that's fine as well.

Leave the `#f00` background override in `crypto.css` though.